docs/guide/testrunner/frameworks.md

```diff
@@ -63,6 +63,10 @@ Make sure to checkout the [transfer promises](/guide/usage/transferpromises.html
 consecutive assertions. If you want to define specific Mocha settings you can do that by adding `mochaOpts`
 to your configuration file.
 
+If you get messages like
+`timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.`
+then you can increasing the `timeout` settings in `mochaOpts`.
+
```

Comment:
I was getting this error and I was unsure what part of the stack it was from, like Mocha, or Webdriver or Selenium. It would happen when I had long running tests that happened remotely and went slowly.